app/i18n/en/conf.php

```diff
@@ -105,6 +105,16 @@
 		'none' => 'None',
 		'small' => 'Small',
 	),
+	'notification_timeout' => array(
+		'bad' => array(
+			'label' => 'Show warning banner',	// TODO
+			'seconds' => 'seconds (minimum 1)',	// TODO
```

Comment:
"minimum 1" doesn't strike me as proper English (I assume it's a bit of a `mindestens 1` sneaking in)
```suggestion
			'seconds' => 'seconds (at least 1)',
```

```suggestion
			'seconds' => 'seconds (a minimum of 1)',
```